code/modules/tooltip/tooltip.dm

```diff
@@ -0,0 +1,113 @@
+/*
+Tooltips v1.0 - 19/10/15
+Developed by Wire (#goonstation on irc.synirc.net)
+- Initial release
+
+Configuration:
+- Set control to the correct skin element (remember to actually place the skin element)
+- Set file to the correct path for the .html file (remember to actually place the html file)
+- Attach the datum to the user client on login, e.g.
+	/client/New()
+		src.tooltips = new /datum/tooltip(src)
+
+Usage:
+- Define mouse event procs on your (probably HUD) object and simply call the show and hide procs respectively:
+	/obj/screen/hud
+		MouseEntered(location, control, params)
+			usr.client.tooltip.show(params, title = src.name, content = src.desc)
+
+		MouseExited()
+			usr.client.tooltip.hide()
+
+Customization:
+- Theming can be done by passing the theme var to show() and using css in the html file to change the look
+- For your convenience some pre-made themes are included
+
+Notes:
+- You may have noticed 90% of the work is done via javascript on the client. Gotta save those cycles man.
+- This is entirely untested in any other codebase besides goonstation so I have no idea if it will port nicely. Good luck!
+	- After testing and discussion (Wire, Remie, MrPerson, AnturK) ToolTips are ok and work for /tg/station13
+*/
+
+
+/datum/tooltip
+	var
+		client/owner
+		control = "mainwindow.tooltip"
+		file = 'code/modules/tooltip/tooltip.html'
+		showing = 0
+		queueHide = 0
+
+
+	New(client/C)
+		if (C)
+			src.owner = C
+			src.owner << browse(src.file, "window=[src.control]")
+
+		..()
+
+
+	proc/show(params = null, title = null, content = null, theme = "default")
```

Comment:
not fixing the pathing, keeping it the same so any future updates from Wirewraith are easier to port.